lib/resque/scheduler/delaying_extensions_old.rb

```diff
@@ -0,0 +1,330 @@
+# vim:fileencoding=utf-8
+require 'resque'
+require_relative 'plugin'
+require_relative '../scheduler'
+
+module Resque
+  module Scheduler
+    module DelayingExtensionsOld
```

Comment:
Do I need to review this thoroughly or is the code the same as before?

Reply:
It's the exact same code. You could do a diff to verify. It could be trimmed down but I didn't want to have to thoroughly justify which methods were removed.